content/200-orm/200-prisma-client/150-using-raw-sql/100-typedsql.mdx

```diff
@@ -0,0 +1,204 @@
+---
+title: 'TypedSQL'
+metaTitle: 'Writing Type-safe SQL with TypedSQL and Prisma Client'
+metaDescription: 'Learn how to use TypedSQL to write fully type-safe SQL queries that are compatible with any SQL console and Prisma Client.'
+sidebar_class_name: preview-badge
+---
+
+## Getting started with TypedSQL
+
+To start using TypedSQL in your Prisma project, follow these steps:
+
+1. Ensure you have `@prisma/client` and `prisma` installed and updated to at least version `5.19.0`.
```

Comment:
Maybe add the command here? npm i prisma@latest @prisma/client@latest (i don't think that's the correct command but you get what I mean)